pkg/services/ngalert/state/persist.go

```diff
@@ -0,0 +1,15 @@
+package state
+
+import (
+	"context"
+
+	"github.com/grafana/grafana/pkg/services/ngalert/models"
+)
+
+type InstanceStore interface {
```

Comment:
You'll notice `GetAlertInstance` is missing - it's dead code aside from tests. There's a task on the linked issue to handle this separately.